### DISCUSSION TITLE
Yellow sign on graphs and search not working (Cloudflare)

### DISCUSSION THREAD

**Discussion by perern** [firefly-iii/firefly-iii#9502] [Firefly III] [answered]
I'm accessing my Docker instance through cloudflare.
I get this sign on a lot of graphs, also search just doesn't do anything.

In the log while searching: 
[26/Nov/2024:14:47:39 +0100] "GET /search?search=Wolt&go= HTTP/1.1" 200 8103 "-" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Mobile Safari/537.36"

How do I enable debugging using docker compose?
I upgraded to latest version yesterday but that didn't help.

**Comment by JC5** [comment, marked as answer]
Hi, thanks for opening a discussion. 

This has happened to a few people before, could you check out the search?

My bet is you're missing TRUSTED_PROXIES=*

**Reply by perern** [reply]
That was it, is it ok with ** or should I be more specific?
If so what IP should I put there?

**Reply by JC5** [reply]
If you want, the internal IP of your reverse proxy or the docker host. But `*` should be enough :)
